gst/mqtt/mqttsink.c

```diff
@@ -930,9 +930,19 @@ gst_mqtt_sink_set_caps (GstBaseSink * basesink, GstCaps * caps)
 
   if (ret && gst_caps_is_fixed (self->in_caps)) {
     gchar *caps_str = gst_caps_to_string (caps);
+    gint res;
```

Comment:
how about use ```gsize```?

Reply:
And I think `len` is better name than `res`.

Reply:
thx updated.